Frontend/src/redux/thunks/userThunks.js

```diff
@@ -0,0 +1,81 @@
+import AsyncStorage from '@react-native-community/async-storage';
+
+const axios = require('axios').default;
+
+import {addUser} from '../actions/userActions';
+
+export const loginUser = ({user, navigation}) => {
+  return async () => {
+    try {
+      const response = await axios.post(
+        `http://192.168.100.2:3000/api/user/login`,
+        user,
+      );
+
+      await AsyncStorage.setItem('accessToken', response.data.accessToken);
+      navigation.replace('TabNavigatorRoutes');
+    } catch (error) {
+      alert(error.response.data.message);
+    }
+  };
+};
+
+export const registerUser = ({user}) => {
+  return async () => {
+    try {
+      await axios.post(`http://192.168.100.2:3000/api/user/register`, user);
+
+      alert('Registration successful.');
+    } catch (error) {
+      alert(error.response.data.message);
+    }
+  };
+};
+
+export const saveUser = () => {
+  return async (dispatch) => {
+    const token = await AsyncStorage.getItem('accessToken');
+
+    try {
+      let user = await axios.get('http://192.168.100.2:3000/api/user', {
+        headers: {
+          authorization: token,
+        },
+      });
+
+      user = user.data.user;
+      user.birthday = user.birthday.substring(0, 10);
+      user.date = user.date.substring(0, 10);
+      dispatch(addUser(user));
+    } catch (error) {
+      alert(error.response.data.message);
+    }
+  };
+};
+
+export const _updateUser = ({
```

Comment:
_ does it have a purpose or is it a mistake?

Reply:
I have another function with the name updateUser in userActions which is used in the same file as this one.